### DISCUSSION TITLE
免费白嫖一个容器服务，并开启v2ray服务

### DISCUSSION THREAD

**Discussion by giscus** [oldwinter/knowledge-garden#58] [主发布站的评论系统]
# 免费白嫖一个容器服务，并开启v2ray服务

另一个白嫖的服务： https://replit.com/@DongdongChen2/vpn
号称永久在线、永久免费。
 Doprax搭建免费的v2ray - 梅塔沃克 - 专注跨境
踩到一个坑，就是给的域名，可能国内dns不通，所以需要直接换成ip，填在客户端配置里。
是德国的ip，ping延时大概500ms，部署完后发现速度很拉胯，100kb每秒。
开始尝试利用[[cloudflare pages]]服务，进行反向代理。
尝试后发现还是直接使用[[Cloudflare Worker]]即可，将vpn2.oldwinter.top的请求全部转发至上面那个被jin的域名，然后客户端的地址记得也改成vpn2.oldwinter.top
客户端配置：
![[Pasted image 20230312131834.png]]

https://garden.oldwinter.top/%E5%85%8D%E8%B4%B9%E7%99%BD%E5%AB%96%E4%B8%80%E4%B8%AA%E5%AE%B9%E5%99%A8%E6%9C%8D%E5%8A%A1%E5%B9%B6%E5%BC%80%E5%90%AFv2ray%E6%9C%8D%E5%8A%A1/

**Comment by ottocsb** [comment]
图挂了
